docs/details/shards/GFX/BuiltinFeature.md

```diff
@@ -0,0 +1,58 @@
+- The `GFX.Feature` shard is used to create a custom feature using the parameters provided while the `GFX.BuiltinFeature` shard is for adding a "ready-made" feature.
```

Comment:
Can you move this leading `-` please from top level? I removed all of them on devel :)